stdlib/public/stubs/Stubs.cpp

```diff
@@ -20,16 +20,25 @@
 #define _WITH_GETLINE
 #endif
 
+#if defined(_MSC_VER)
+#include <string>
+#include <iostream>
+#include <windows.h>
+// Macro max(), min() conflicts with std::max(), std::min()
```

Comment:
Please `#define NOMINMAX` instead (per https://support.microsoft.com/en-us/kb/143208) -- here and in other files.